src/App.js

```diff
@@ -19,8 +25,7 @@ const App = () => (
         <Switch>
           <Route path="/signin" exact component={Signin} />
           <Route path="/social/auth/success" exact component={SocialLogin} />
-          <Route path="/dashboard" exact component={Dashboard} />
-          <Route path="/dashboard" exact component={VerifyEmail} />
```

Comment:
@NiyoEric why did you remove this Route.

Reply:
Let me add it again.

Reply:
Try to start a local server again and visit `http://localhost:4000/dashboard`.